plugins/br.com.embraer.massif.commandevaluation.commands/.project

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>br.com.embraer.massif.commandevaluation.commands</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.pde.ManifestBuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.pde.SchemaBuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.sonarlint.eclipse.core.sonarlintBuilder</name>
```

Comment:
Do we use SonarLint for Massif? In my experience, the use of SonarLint slows down compiling very noticeably, so if we don't plan to set it up, we might leave it out from here.

Reply:
I suppose Péter only added it to keep it consistent with all other places. I agree that we can remove it from the project configurations. However, it doesn't have to be done as part of the PR.